cli/foundation-models/system/distillation/conversation/README.md

```diff
@@ -0,0 +1,24 @@
+# Distillation with CLI (Conversation)
+
+## 1. Create the Job
```

Comment:
Can you have a link for SDK notebook for the .jsonl creation reference?

Reply:
Added note beneath this section to see the notebook example for the reference